src/scss/includes/panels/poi_panel.scss

```diff
@@ -1,3 +1,5 @@
+@import "../osm_contribute";
```

Comment:
import "..."; OR import '...'

Reply:
i don't fix automatic import syntax for now